### PR TITLE
CXP-1100: Clean Apps infrastructure folder in Connectivity

### PR DESCRIPTION
**Pull request by willy-ahva**
**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

**Review by guillaumefer-akeneo — APPROVED** [review]

File: src/Akeneo/Connectivity/Connection/back/Infrastructure/Apps/Controller/ExternalApi/GetWizardDataAction.php [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 declare(strict_types=1);
 
-namespace Akeneo\Connectivity\Connection\Infrastructure\Apps\Controller;
+namespace Akeneo\Connectivity\Connection\Infrastructure\Apps\Controller\ExternalApi;
```
Comment (willy-ahva) [inline src/Akeneo/Connectivity/Connection/back/Infrastructure/Apps/Controller/ExternalApi/GetWizardDataAction.php:5]: internal

**Comment by willy-ahva** [comment]
InternalApi -> Internal
ExternalApi -> Public

**Comment by willy-ahva** [comment]
Split InstallSubscriber

**Comment by willy-ahva** [comment]
Install query user consent dans app

**Comment by willy-ahva** [comment]
split config.yml

File: src/Akeneo/Connectivity/Connection/back/Infrastructure/Symfony/Resources/config/Apps/routing/apps.yml [hunk 2]
```diff
@@ -1,49 +1,49 @@
 akeneo_connectivity_connection_apps_rest_get_app_activate_url:
```
Comment (masacc) [inline src/Akeneo/Connectivity/Connection/back/Infrastructure/Symfony/Resources/config/Apps/routing/apps.yml:1]: rename this file : `config/Apps/routing.yml`

File: src/Akeneo/Connectivity/Connection/back/tests/PhpStan/Rule/FinalControllerRule.php [hunk 3]
```diff
@@ -10,7 +10,7 @@
 
 final class FinalControllerRule extends AbstractControllerRule
 {
-    private const ERROR_MESSAGE = 'Controller must be final';
+    private const ERROR_MESSAGE = 'ExternalApi must be final';
```
Comment (masacc) [inline src/Akeneo/Connectivity/Connection/back/tests/PhpStan/Rule/FinalControllerRule.php:13]: nope

**Review by masacc — APPROVED** [review]

**Pull request merged**